### ISSUE TITLE
Adjust shebang line?

### ISSUE THREAD

**Issue by mattmccray** [mweststrate/nscript#6]
The default shebang `#!/usr/bin/nscript` failed to work for me. Perhaps because I use NVM on a Mac, I'm not sure. But changing it to use `env` worked nicely:

```
#!/usr/bin/env nscript
```

**Comment by mweststrate** [comment]
Good point, that is a better approach indeed. Thanks for the suggestion
